internal/services/databricks/validate/cidr.go

```diff
@@ -0,0 +1,21 @@
+package validate
+
+import (
+	"fmt"
+	"net"
+)
+
+// Evaluates if the passed CIDR is a valid IPv4 or IPv6 CIDR.
+func CIDR(input interface{}, key string) (warnings []string, errors []error) {
```

Comment:
we should call out this is both IPv4 and IPv6:

```suggestion
func CIDRIsIPv4OrIPv6(input interface{}, key string) (warnings []string, errors []error) {
```

Reply:
Fixed.